### DISCUSSION TITLE
Ollama models

### DISCUSSION THREAD

**Discussion by olimorris** [olimorris/codecompanion.nvim#38] [General]
As someone that doesn't use Ollama a great deal, I'm keen to get any input from the community around what the default models should be.

Thanks to @lazymaniac for their [feedback](https://github.com/olimorris/codecompanion.nvim/pull/32#issuecomment-2028886439) thus far and I've incorporated this into the adapter.

**Comment by ryoppippi** [comment]
For me, 
- `qwen2.5-coder`
- `codestral:22b-v0.1-q4_K_S`

They are really good.

For default, I suggest `qwen2.5-coder` because this model is not so large

**Reply by hampgoodwin** [reply]
I'd also now add deepseek-coder-v2, and aya-expanse into the mix. Both have worked well for me with general chat/coding questions. I am still getting some mixed results with tools aspects of the models, though.
